OpenEdXMobile/src/main/java/org/edx/mobile/event/AppUpdatedEvent.java

```diff
@@ -2,6 +2,10 @@
 
 import android.support.annotation.NonNull;
 
+/**
+ * A sticky event fired on application creation time if app has been updated to a newer version
+ * on device
```

Comment:
The event being sticky is an implementation detail and should not be included in the javadoc.

A better javadoc would be:
`An event fired on application start incase the app updated to a newer version.`